1_3.py

```diff
@@ -0,0 +1,46 @@
+"""
+Autor: Voitov Vladimir
+Date: 16.11.2021
+Task: Class Work for L3
+"""
```

Comment:
This style prefere! Try to use it in the next home works

#######################
#Author: 
#Date: 
#Task:
#######################